client/app.jsx

```diff
@@ -23,12 +39,67 @@ const App = () => {
   }, []);
 
   useEffect(() => {
-    window.addEventListener('hashchange', event => {
-      setRoute(parseRoute(window.location.hash));
-    });
-  });
+    const token = window.localStorage.getItem('react-context-jwt');
+    setUser(token ? jwtDecode(token) : null);
+    setAuth(false);
+  }, []);
+
+  useEffect(() => {
+    fetch('/api/userList')
+      .then(res => res.json())
+      .then(retrievedList => setUserList(retrievedList));
+  }, []);
+
+  const handleSignIn = result => {
+    const { user, token } = result;
+    window.localStorage.setItem('react-context-jwt', token);
+    setUser(user);
+  };
+
+  const handleSignOut = () => {
+    window.localStorage.removeItem('react-context-jwt');
```

Comment:
After updating your token name make sure to change it everywhere you are accessing the token from `localStorage`